src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -654,6 +666,25 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 " `low_cpu_mem_usage=False`."
             )
 
+        if device_map is not None and not is_accelerate_available():
+            raise NotImplementedError(
+                "Using `device_map` requires the `accelerate` library. Please install it using: `pip install accelerate`."
+            )
```

Comment:
@SunMarc I think it could be nice to make the `accelerate` version even more specific here so that we can benefit from the warning fixes you recently shipped?

Reply:
Cool! Just ping me once that is done and I will update accordingly :)